src/client/Client.cc

```diff
@@ -12291,6 +12291,7 @@ int Client::ll_get_stripe_osd(Inode *in, uint64_t blockno,
   uint32_t object_size = layout->object_size;
   uint32_t su = layout->stripe_unit;
   uint32_t stripe_count = layout->stripe_count;
+  assert(stripe_count);
```

Comment:
it might be better to guard the divide with a conditional.  the cluster shouldn't have this zeroed field, but if it does, we don't need to crash the client.